src/courseware/CoursewareContainer.jsx

```diff
@@ -310,6 +329,7 @@ class CoursewareContainer extends Component {
           nextSequenceHandler={this.handleNextSequenceClick}
           previousSequenceHandler={this.handlePreviousSequenceClick}
           unitNavigationHandler={this.handleUnitNavigationClick}
+          sidebarNavigationClickHandler={this.handleOutlineSidebarNavigationClick}
```

Comment:
Why do you need to pass down this property? Can't you use it directly in the Sequence component? @anfbermudezme